lib/query-builders/inspector.js

```diff
@@ -24,9 +37,12 @@ module.exports = {
       ntcoEndorsed.id,
       withLicensing.id,
       inspectorRecommended.id,
-      inspectorRejected.id,
-      returnedToApplicant.id
-    ]);
+      inspectorRejected.id
+    ])
+      .orWhere(builder => {
+        builder.where('status', returnedToApplicant.id)
+          .whereJsonNotSupersetOf('data', { changedBy: profile.id });
+      });
```

Comment:
I'm not sure this bit is correct? It seems like inspectors will see all tasks that are returned to applicants and _not_ their own i.e. everything that is returned to _someone else_.

I suspect `whereJsonNotSupersetOf` should be `whereJsonSupersetOf`.

Reply:
Having said that, I don't think this is necessary at all. Inspectors seeing returned to applicant things is handled by the "applicant" query builder.

(unless this is intended to do something else that I don't understand)

Reply:
the latter, we're showing all returned to applicant tasks here (in progress) that are _not_ the inspector's tasks

Reply:
it may not be the right way of doing it, but without some kind of check the returned to applicant task when you are an inspector and the applicant, appears in my tasks, outstanding, _and_ in progress

Reply:
Ahhh, the collapsed code threw me. I'd read it as being part of the `outstanding` query.

Carry on...